File: 00. Estructura/01. Lenguajes de proposito general/02. Programación básica/07. Funciones/07. Lambda (Literal o anónima)/JavaScript/Lambda.js

// Creación con parametros limitados
funcion = (var1, var2, varN) => {
	// Contenido
}

// Creación con parametros ilimitados
funcion = (...variables) => {
	// Contenido
}
       
// Creación con parametros por defecto (Los valores por defecto de pueden omitir en el llamado)
funcion = (var1 = def1, var2 = def2, varN = defN) => {
	// Contenido
}

// Llamado
/* Contexto */ funcion(valor1, valor2, valorN) // Contexto

// Eliminación
NO SOPORTADO (EL GARBAGE COLLECTOR LA ELIMINA AUTOMATICAMENTE CUANDO YA ES INACCESIBLE)

File: 01. Lenguajes de proposito general/02. Programación básica/07. Funciones/07. Lambda (Literal o anónima)/Python/Lambda.py

# Creación con parametros limitados
funcion = lambda var1, var2, varN: expresión_retorno

# Creación con parametros ilimitados
funcion = lambda *variables: expresión_retorno
       
# Creación con parametros por defecto (Los valores por defecto de pueden omitir en el llamado)
funcion = lambda var1 = def1, var2 = def2, varN = defN: expresión_retorno

# Llamado
''' Contexto ''' funcion(valor1, valor2, valorN) # Contexto

# Eliminación
del funcion
